libs/frontend/feature-build/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -167,7 +167,9 @@ export const BuilderDndComponent = defineComponent({
               onDrop: dnd.drop,
               onDragend: dnd.dragend,
             }
-          : undefined
+          : {
+              onDragend: dnd.dragend,
+            }
```

Comment:
This is for [#854](https://github.com/samatechtw/pubstudio/issues/854).

The cause of this bug is that even though we forbid a component from being dragged when its ProseMirror editor is active, its parent can still be dragged at the same time. Since a ProseMirror editor is still active when the parent is dropped, the `onDragend` event will not be attached to the component, leading to DND state not being cleared on drop.

So in short, we should attach `onDragend` event to DND component all the time so that DND state can be cleared on drop.